hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/webapp/NodesBlock.java

```diff
@@ -81,6 +82,7 @@ protected void render(Block html) {
       int usedMemory = (int) info.getUsedMemory();
       int availableMemory = (int) info.getAvailableMemory();
       TR<TBODY<TABLE<Hamlet>>> row = tbody.tr();
+      row.td().__(info.getClusterId());
```

Comment:
Can you attach a screenshot with the end result?

Reply:
Sorry, I lost the screenshot